javalib/src/main/scala/scala/scalanative/nio/fs/unix/UnixFileSystemProvider.scala

```diff
@@ -24,7 +24,7 @@ class UnixFileSystemProvider extends GenericFileSystemProvider {
   }
 
   private def getUserDir(): String = {
-    val buff = stackalloc[CChar](4096.toUInt)
+    val buff: Ptr[CChar] = stackalloc[CChar](4096.toUInt)
```

Comment:
And sometimes you use it for `stackalloc` and sometimes not.

Reply:
I've use it for all usages of `stackalloc[T](n)`, for `stackalloc[T]()` it's not needed. If it's not used somewhere then its probably my mistake and explicit type should be added there